MiniApp/Classes/RealMiniApp.swift

```diff
@@ -66,8 +66,8 @@ internal class RealMiniApp {
             } }
     }
 
-    func createMiniApp(appId: String, completionHandler: @escaping (Result<MiniAppDisplayProtocol, Error>) -> Void, messageInterface: MiniAppMessageDelegate? = nil) {
-        getMiniApp(miniAppId: appId) { (result) in
+    func createMiniApp(appId: String, version: String? = nil, completionHandler: @escaping (Result<MiniAppDisplayProtocol, Error>) -> Void, messageInterface: MiniAppMessageDelegate? = nil) {
```

Comment:
I think it's okay for now, but an improvement for the future would be caching the files for multiple versions in preview mode. 

On Android, the downloader will check if it's in preview mode and only delete the cached version if it's not. So in published mode, it can only have one version downloaded at a time, but in preview mode it can have many versions downloaded. See here: https://github.com/rakutentech/android-miniapp/blob/f5f4c7d40751ba9416be88ec2018296a031d3735/miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniAppDownloader.kt#L115

Reply:
I've done something similar in a separate PR #175